src/app/windows/main/menus/option/about/about.component.html

```diff
@@ -4,7 +4,7 @@
 
         <div class="col-xs-12">
 
-            <h2>{{ applicationService.appName }} {{ version }}</h2>
+            <h2>{{ applicationService.appName }} <!-- {{ version }} --> Custom 2.6</h2>
```

Comment:
I'm not sure we will keep this version =)

Reply:
Ahah just what i say in discord, i get the code from my custom repo xD